tgui/packages/tgui/interfaces/CheckAntags.js

```diff
@@ -0,0 +1,331 @@
+                minValue={1}
+                maxValue={100}
+                value={midround_living_limit*100}
+                onChange={(e, value) => act('living_crew_limit', {
+                  crewlimit: value,
+                })} /> : ''}{continue1 && continue2 ? '% of crew alive' : ''}
+              <br />
+              {continue1 && continue2 ? 'If limits past: ' : ''}
+              {continue1 && continue2 ? <Button
+                content={end_on_death_limits ? 'End The Round' : 'Continue As Extended'}
+                onClick={() => act(end_on_death_limits ? 'cont_death_limits' : 'end_death_limits')} /> : ''}
+            </Section>
+          </Grid.Column>
+        </Grid>
+        <br />
+        <Grid height={38.5}>
+          <Grid.Column>
+            <Section title="Game Manipulation" fill>
+              <Button.Confirm
+                content="End Round Now"
+                onClick={() => act('end_round')} /> <br />
+              <Button
+                content="Delay Round End"
+                onClick={() => act('delay_round')} /> <br />
+              <Button
+                content="Enable/Disable CTF"
+                onClick={() => act('toggle_ctf')} /> <br />
+              <Button.Confirm
+                content="Reboot World"
+                onClick={() => act('trigger_reboot')} /> <br />
+              <Button
+                content="Check Teams"
+                onClick={() => act('check_teams')} /> <br />
+              <Divider />
+              <Box bold color="blue">Players</Box>
+              <Box color="blue">{connected_players - lobby_players} ingame <br /> {connected_players} connected <br /> {lobby_players} in lobby</Box>
+              <br />
+              <Box bold color="green">Living Players</Box>
+              <Box color="green">{living_players_connected} active <br /> {living_players - living_players_connected} disconnected <br /> {living_players_antagonist} antagonists</Box>
+              <br />
+              <Box bold color="pink">SKIPPED [CC Z-level]</Box>
+              <Box color="pink">{living_skipped} living players <br /> {drones} living drones</Box>
+              <br />
+              <Box bold color="red">Dead/Observing Players</Box>
+              <Box color="red">{observers_connected} active <br /> {observers - observers_connected} disconnected <br /> {brains} brains</Box>
+              { other_players ? { other_players }+"invalid players!" : "" }
+              <br />
+              <Button
+                content="Legacy Panel"
+                onClick={() => act("forcelegacy")} />
+            </Section>
+          </Grid.Column>
+          <Grid.Column size={3}>
+            <Section title="Antagonists" fill>
+              {priority_sections.map((teamdata, index) =>
+                (
+                  <Section title={teamdata[0]} key={teamdata.index}>
+                    {teamdata[1].map((antagdata, idx) =>
```

Comment:
when you've redone the DM side as associative lists these will function like objects